sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -37,6 +37,9 @@ private[sql] case class JDBCPartitioningInfo(
     numPartitions: Int)
 
 private[sql] object JDBCRelation {
+
+  import scala.collection.JavaConverters._
```

Comment:
import scala.collection.JavaConverters._ moved to the top with other non-wildcard imports